server/controllers/v1/bookController.js

```diff
@@ -0,0 +1,13 @@
+import Book from '../../models/book';
+
+export const addBook = (req, res) => {
+    try {
+        const book = new Book(req.body);
+        book.save();
+        return res.status(201).json({ message: `Book with ${book.title} has been added`, book })
+    } catch (error) {
+        console.log(error);
```

Comment:
Why are you logging error on this line?

Reply:
It was an over sight. I have removed it in  my updated file